client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -484,3 +484,29 @@
 		display: none;
 	}
 }
+
+// Fix small CSS with payment modal. Ideally, should
+// be resolved via css for payment modal component itself.
+// But that modal doesn't have its own css. The CSS is part
+// of the Earn > Memberships page.
```

Comment:
Instead of copying the CSS here, can you move the CSS to a new file and import from `my-sites/earn/memberships/add-edit-plan-modal`? This is fine for merging but would need a follow-up to restructure.

Restructuring will be helpful for future with this modal as well.

Reply:
Yes, I thought about doing that here, but I didn't want to mix the point of this PR with refactoring the payments modal component to have it's own CSS. I can handle in a follow up.

Reply:
Yup that works! 👍